validation/validation_runner.py

```diff
@@ -139,9 +140,9 @@ def comparison_job(
         job.depends_on(dependency)
 
     job.image(get_config()['image']['happy'])
-    job.memory('20Gi')
-    job.storage('40Gi')
-    job.cpu(2)
+    job.memory('100Gi')
```

Comment:
that is hungry.

Reply:
Yeah, I think 100 might be over-doing it, but the mutil-region stratification is insanely greedy. I provisioned 70 and it used 80+, and these are relatively small single sample VCFs